docs/team/community-strategy.md

```diff
@@ -0,0 +1,24 @@
+# Community Strategy
+
+This page contains information regarding activities undertaken and community strategies developed by the Community Strategic Lead.
+[Sarah Gibson](https://github.com/sgibson91) is currently serving in this role as of August 2022, for a period of two years.
+
+This role and related activities are funded by a CZI EOSS DEI grant, administered by NumFOCUS.
+
+```{admonition} Work in Progress
+The work represented on this page is under active development and we will update information as soon as we are able to as we iterate.
+```
+
+Some information is being shared in [Issue #536](https://github.com/jupyterhub/team-compass/issues/536), which will be migrated here over time.
```

Comment:
The PyData theme will soon release support for lightly auto-formatting github issue/PR links, so we could just put in the raw link if we like: https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/theme-elements.html#link-shortening-for-git-repository-services